hometask 4.py

```diff
@@ -0,0 +1,51 @@
+phonebook = {}
+
+name = input('Enter a name:')
+
+number = input('Enter a number:')
+
+
+if phonebook.get(name):
+    print(f'{name} already exists. You need to delete a record and add again if you want to change it')
+else:
+    phonebook[name] = number
+
+
+
+
+while True:
+    user_input = input('Enter a command:')
+    split_input = user_input.split()
+
+    command = split_input[0]
+    if command == 'stats':
+        print(len(phonebook))
+    elif command == 'add':
+        name=input('Enter a name:')
+        if name in phonebook.keys():
+            print("Fault. There is such name in a phonebook.")
+
+        else:
+            print("There is no such name in a phonebook. Continue.")
+
+        number=input('Enter a number:')
+        phonebook[name]=number
```

Comment:
There shouldn't exist an ability to override records. Your check above just informs that the name is already in phonebook, but then you anyway override it. Move it inside of `else` condition